Core/clim-core/commands.lisp

```diff
@@ -223,6 +223,29 @@ designator) inherits menu items."
 
 (make-command-table 'user-command-table)
 
+<<<<<<< HEAD
```

Comment:
this is invalid syntax for common lisp (probably a result of merge conflicts). generally speaking please rather rebase pull requests instead of merging them - that will make peer review easier.